### ISSUE TITLE
username & password in server-settings.json CPU 100% no server start

### ISSUE THREAD

**Issue by Omaha2002** [pdemonaco/factorio-mod-updater#21]
If I put my credentials in server-settings.json as instructed all modules get checked and updated but the server won't start anymore after stopping and starting the factorio-headless server.

![image](https://user-images.githubusercontent.com/3179748/97996993-c5052900-1de8-11eb-91e2-b8e4aef0c3d4.png)
![image](https://user-images.githubusercontent.com/3179748/97997344-2927ed00-1de9-11eb-8781-d663427ba286.png)

CPU spikes to 100%, as soon as I remove username and password from server-settings, not even stopping the server, CPU turns to normal and I'm able to connect to game.

![image](https://user-images.githubusercontent.com/3179748/97997615-71470f80-1de9-11eb-8f57-a6fcb8232a77.png)

Ofcourse omitting username password gives an error while updating:

![image](https://user-images.githubusercontent.com/3179748/97996880-98511180-1de8-11eb-8c57-7c7b45fbaf35.png)

**Comment by Omaha2002** [comment]
in the factorio-current.log log I see this:

   7.535 Prototype list checksum: 1281798343
   7.543 Verbose ModManager.cpp:554: Time to load mods: 7.53721
   7.675 Info PlayerData.cpp:68: Local player-data.json available, timestamp 1604413939
   7.675 Info PlayerData.cpp:75: Cloud player-data.json unavailable
   7.676 Info GlobalContext.cpp:1269: Resetting config.
   7.678 Factorio initialised
   7.678 Error CommandLineMultiplayer.cpp:88: Only one of token or password can be set at once.
   7.682 Info ServerMultiplayerManager.cpp:136: Quitting multiplayer connection.
   7.682 Info ServerMultiplayerManager.cpp:769: updateTick(4294967295) changing state from(Ready) to(Closed)
   7.759 Goodbye

Stating only one token or password can be set, so my bad, I typed password AND token, I removed password and update is working.

Fixed!
